californium-core/pom.xml

```diff
@@ -136,7 +149,6 @@
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-jar-plugin</artifactId>
-				<version>3.0.2</version>
```

Comment:
f we remove the version here, we should declare it in the top level POM in build/pluginManagement. Otherwise we might use different versions of the plugin in different builds.

Reply:
OK